e2e/Dockerfile

```diff
@@ -32,6 +32,7 @@ ARG OSTYPE=Linux
 ARG USER=circleci
 ARG HOME=/home/$USER
 ARG APP=$HOME/repo
+ARG DOCKER=true
```

Comment:
hey, @sfc-gh-tszerszen , what is a purpose of adding `DOCKER=true` here?

Reply:
I second this question. I though I was lacking some context here but also was wondering why are we have configuration changes within this PR.

Reply:
AFAIK now we don't use this Dockerfile to run e2e tests on CI (since we migrated from CircleCI to Github Actions in our CI).  This probably means we are no longer tied to the `circleci/python:3.7.11` image and could use anything that is more convenient for running tests locally with Docker (on our CI we use GitHub actions ubuntu image)

in my humble opinion, this could be a great separate project to optimize our e2e tests flow with Docker locally, and get rid of obsolete CircleCI artifacts in our codebase, I think it's worth discussing with @vdonato

Reply:
Sure, but can we merge it for now, because the E2E tests generated locally by our old Dockerfile are still useful and relevant when developing features locally, so it's definitely a small improvement